python-selenium/README.md

```diff
@@ -0,0 +1,42 @@
+# Modern Web Automation With Python and Selenium
+
+This repository contains the module `bandcamp`, which is the sample app built in the Real Python tutorial [Modern Web Automation With Python and Selenium](https://realpython.com/modern-web-automation-with-python-and-selenium/).
+
+## Installation and Setup
+
+Create and activate a [Python virtual environment](https://realpython.com/python-virtual-environments-a-primer/).
+
+Then, install the requirements:
+
+```sh
+(venv) $ python -m pip install -r requirements.txt
+```
+
+The only direct dependency for this project is [Selenium](https://selenium-python.readthedocs.io/). You should use a Python version of at least 3.8.
+
+## Run the Bandcamp Discover Player
+
+To run the music placer, navigate to the `src/` folder, then execute the module from your command-line:
+
+```sh
+(venv) $ cd src/
+(venv) $ python -m bandcamp
```

Comment:
:wrench: **Technical Recommendation**

That doesn't work:

```
(python-selenium) $ cd src/
(python-selenium) $ python -m bandcamp/
/home/bartek/.virtualenvs/python-selenium/bin/python: No module named bandcamp/
```

A better option would be to install the package into the venv first. That way, it won't matter where you'll run the module from:

```
(python-selenium) $ python -m pip install .
(python-selenium) $ python -m bandcamp/
```